examples/traits/mount/README.md

```diff
@@ -0,0 +1,17 @@
+# Camel K Mount Trait
+
+
+In this section you will find examples about fine tuning your `Integration` using **Mount** `trait` capability.
+
+
+**Example 1: Mount Persistent Volume Claim on the Integration Pod in a Kubernetes platform (Static Volume Provisioning)**
+
+Requirement: Persistent Volume already created.
+
+Create the Pod and the Persistent Volume Claim:
+`kubectl apply -f pod-example.yaml`
```

Comment:
You don't need any live Pod for the PVC to be usable. You can just remove this line.

Reply:
done!